docs/_embed/partner_api.md

```diff
@@ -140,11 +140,15 @@ The idea is to use an intermediate page that reads the access token from the URL
 
 Available parameters to the Apps resource:
 
-| parameter     | requirement | notes                                                                                  |
-| ------------- | ----------- | -------------------------------------------------------------------------------------- |
-| **client_id** | Required    | Your application client ID.                                                            |
-| **per_page**  | Optional    | (defaults to 100, max of 100) Limit the number of apps returned.                       |
-| **page**      | Optional    | (defaults to 1) The page number. Page number 1 refers to the first page in the result. |
+| parameter                   | requirement | notes                                                                                         |
+| --------------------------- | ----------- | --------------------------------------------------------------------------------------------- |
+| **client_id**               | Required    | Your application client ID.                                                                   |
+| **category**                | Optional    | Filter the results by app category.                                                           |
+| **is_in_zap_template_with** | Optional    | Service slug that limits the results to apps that are included within the same Zap Templates. |
```

Comment:
@cyberwitch Did we change the name of this to `shares_zap_templates` recently?

Reply:
Oops, that's an old name. Will update it - thanks for spotting the discrepancy!